src/shared-components/button/shared-components/loader/style.ts

```diff
@@ -13,7 +13,9 @@ const statefulLoader = keyframes`
 `;
 
 const primaryLoadingStyles = (theme: ThemeType) => `
-  background-color: ${theme.COLORS.white};
+  background-color: ${
+    theme.__type === 'primary' ? theme.COLORS.white : theme.COLORS.primary
```

Comment:
If we end up having more exceptions like this, we might want to create a separate file that contains helper methods that contains all the exceptions (e.g. `primaryButtonLoadingBackgroundColor = (theme) => {}` and then embed it). It might be worth clarifying with Ben if we expect more, or adding a TODO for tracking purposes, but this works for now!

Reply:
Yeah that's a good idea, we have very cases in Radiance but we can do as preparation for PocketDerm considering C8 will introduce a bunch of those in Curology